crates/trade/src/cfd.rs

```diff
@@ -3,6 +3,8 @@ use bdk::bitcoin::Denomination;
 use rust_decimal::prelude::ToPrimitive;
 use rust_decimal::Decimal;
 
+pub const BTCUSD_MAX_PRICE: u64 = 1_048_575;
```

Comment:
nit: I guess we should add a comment here, that this number is not chosen randomly, but is actually a requirement coming from the p2pdoracle we are using at the moment.

🤔 one place were we could put this constant could be on the oracle client, if we choose to wrap it.